scripts/qa_crowbarsetup.sh

```diff
@@ -2043,6 +2045,21 @@ function onadmin_post_allocate()
             ssh $node "getent passwd glance >/dev/null || \
                 useradd -r -g glance -u 450 -d /var/lib/glance -s /sbin/nologin -c \"OpenStack glance Daemon\" glance"
         done
+
+        if iscloudver 6plus && [[ $want_sbd = 1 ]] ; then
+            zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python/$slesdist/ --non-interactive install python-sh
+            chmod +x iscsictl.py
+            ./iscsictl.py --service target --host $(hostname) --no-key
+
```

Comment:
Added cluster and clustername; node is already set at the beginning of the function